nav2_util/include/nav2_util/nav2_clients/clear_except_costmap_service_client.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019 Intel Corporation
```

Comment:
I don't like the idea of adding additional utilities no application utilizes. I think keeping these on the back burner until there's something using it is best